scripts/us_census/acs5yr/subject_tables/common/testdata/colMapInputs/spec.json

```diff
@@ -0,0 +1,240 @@
+		"Estimate!!Total Uninsured!!EDUCATIONAL ATTAINMENT!!Civilian noninstitutionalized population 25 years and over": ["Less than high school graduate", "High school graduate (includes equivalency)", "Some college or associate's degree", "Bachelor's degree or higher"],
+		"Estimate!!Total Uninsured!!EMPLOYMENT STATUS!!Civilian noninstitutionalized population 16 years and over": ["In labor force", "Employed", "Unemployed", "Not in labor force"],
+		"Estimate!!Total Uninsured!!WORK EXPERIENCE!!Civilian noninstitutionalized population 16 to 64 years": ["Worked full-time, year round in the past 12 months", "Worked less than full-time, year round in the past 12 months", "Did not work"],
+		"Estimate!!Total Uninsured!!Civilian noninstitutionalized workers 16 years and over": ["Private for-profit wage and salary workers", "Employee of private company workers", "Self-employed in own incorporated business workers", "Private not-for-profit wage and salary workers", "Local government workers", "State government workers", "Federal government workers", "Self-employed workers in own not incorporated business workers", "Unpaid family workers", "Agriculture, forestry, fishing and hunting, and mining", "Construction", "Manufacturing", "Wholesale trade", "Retail trade", "Transportation and warehousing, and utilities", "Information", "Finance and insurance, and real estate and rental and leasing", "Professional, scientific, and management, and administrative and waste management services", "Educational services, and health care and social assistance", "Arts, entertainment, and recreation, and accommodation and food services", "Other services (except public administration)", "Public administration", "Sales and office occupations", "Natural resources, construction, and maintenance occupations", "Production, transportation, and material moving occupations"],
+		"Estimate!!Total Uninsured!!EARNINGS IN THE PAST 12 MONTHS (IN 2019 INFLATION ADJUSTED DOLLARS)!!Civilian noninstitutionalized population 16 years and over with earnings": ["$1 to $4,999 or loss", "$5,000 to $14,999", "$15,000 to $24,999", "$25,000 to $34,999", "$35,000 to $49,999", "$50,000 to $74,999", "$75,000 or more"],
+		"Estimate!!Total Uninsured!!HOUSEHOLD INCOME (IN 2019 INFLATION ADJUSTED DOLLARS)!!Total household population": ["Under $25,000", "$25,000 to $49,999", "$50,000 to $74,999", "$75,000 to $99,999", "$100,000 and over"],
+		"Estimate!!Total Uninsured!!RATIO OF INCOME TO POVERTY LEVEL IN THE PAST 12 MONTHS!!Civilian noninstitutionalized population for whom poverty status is determined": ["Below 50 percent of the poverty level", "50 to 99 percent of the poverty level", "100 to 149 percent of the poverty level", "150 to 199 percent of the poverty level", "200 to 299 percent of the poverty level", "At or above 300 percent of the poverty level"]
```

Comment:
Could we put the array value per line to make it more readable?

Reply:
Updated :)